controller/ingress_entry.go

```diff
@@ -60,14 +60,14 @@ func (entry LoadBalancerEntry) ValidateEntry() bool {
 }
 
 // SortedByName returns the update with entries ordered by their Name.
-func (u LoadBalancerUpdate) SortedByName() LoadBalancerUpdate {
-	sortedEntries := make([]LoadBalancerEntry, len(u.Entries))
+func (u IngressUpdate) SortedByName() IngressUpdate {
+	sortedEntries := make([]IngressEntry, len(u.Entries))
 	copy(sortedEntries, u.Entries)
 	sort.Sort(byName(sortedEntries))
-	return LoadBalancerUpdate{Entries: sortedEntries}
+	return IngressUpdate{Entries: sortedEntries}
 }
 
-type byName []LoadBalancerEntry
+type byName []IngressEntry
 
 func (a byName) Len() int           { return len(a) }
```

Comment:
This is required for sorting the entries. To sort something you have to implement `Len` and `Swap`. That was done in a separate PR when I ordered nginx entries.